README.md

```diff
@@ -1 +1,142 @@
-# cluster-api-provider-digitalocean
\ No newline at end of file
+# Kubernetes cluster-api-provider-digitalocean Project [![Build Status](https://circleci.com/gh/kubermatic/cluster-api-provider-digitalocean.svg?style=shield)](https://circleci.com/gh/kubermatic/cluster-api-provider-digitalocean/) [![Go Report Card](https://goreportcard.com/badge/github.com/kubermatic/cluster-api-provider-digitalocean)](https://goreportcard.com/report/github.com/kubermatic/cluster-api-provider-digitalocean)
+
+This repository hosts a concrete implementation of a provider for [DigitalOcean](https://www.digitalocean.com/) for the [cluster-api project](https://github.com/kubernetes-sigs/cluster-api).
+
+## Project Status
+
+This project is currently Work-in-Progress and may not be production ready. There is no backwards-compatibility guarantee at this point.
+Checkout the Features portion of the README for details about the project status.
+
+## Getting Started
+
+### Prerequisites
+
+In order to create a cluster using `clusterctl`, you need the following tools installed on your local machine:
+
+* `kubectl`, which can be done by following [this tutorial](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [`minikube`](https://kubernetes.io/docs/tasks/tools/install-minikube/) and the appropriate [`minikube` driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md). We recommend `kvm2` for Linux and `virtualbox` for macOS
+* [DigitalOcean API Access Token generated](https://www.digitalocean.com/docs/api/create-personal-access-token/) and set as the `DIGITALOCEAN_ACCESS_TOKEN` environment variable,
+* Go toolchain [installed and configured](https://golang.org/doc/install), needed in order to compile the `clusterctl` binary,
+* `cluster-api-provider-digitalocean` repository cloned:
+```bash
+git clone https://github.com/kubermatic/cluster-api-provider-digitalocean $GOPATH/src/github.com/kubermatic/cluster-api-provider-digitalocean
+```
+
+### Building `clusterctl`
+
+The `clusterctl` tool is used to bootstrap an Kubernetes cluster from zero. Currently, we have not released binaries, so you need to compile it manually.
+
+Compiling is done by invoking the `compile` Make target:
+```bash
+make compile
+```
+
+This command generates three binaries: `clusterctl`, `machine-controller` and `cluster-controller`, in the `./bin` directory. In order to bootstrap the cluster, you only need the `clusterctl` binary.
+
+The `clusterctl` can also be compiled manually, such as:
+```bash
+cd $GOPATH/github.com/kubermatic/cluster-api-provider-digitalocean/clusterctl
```

Comment:
Replace `$GOPATH` with `$(go env GOPATH)` as $GOPATH does not need to be set in Go anymore since it has a default of `~/go`